feature_details.md

```diff
@@ -0,0 +1,174 @@
+
+* The container size for the element animates from the paint bounds of the previous element to the paint bounds of the new element. During this animation, neither element’s content fits the size of the container. Fitting the painted content within the container is similar to a resize operation to fit a [replaced element’s](https://developer.mozilla.org/en-US/docs/Web/CSS/Replaced_element) content. The default animation uses the behaviour defined by fill mode in [object-fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) but customizing this should be supported going forward. TODO : Add example.
+
+* The pixels from the painted content of the 2 elements are blended using a [cross-fade](https://developer.mozilla.org/en-US/docs/Web/CSS/cross-fade()) animation from `cross-fade(previous 100%, new 0%)` to `cross-fade(previous 0%, new 100%)`.
+
+It’s conceivable that support for animatable properties will be added incrementally and not all will be supported across UAs. This is still an open question and one option could be for the API to allow feature detection to detect which properties are supported by the UA. Using an unsupported property on a shared element has undefined behaviour.
+
+#### Animation Configuration
+The following fields are used in a dictionary, referred to as `ElementTransition`, to configure a single enter/exit or paired transition :
+
+* `transitionType` : This is an enum to provide a predefined set of animation patterns for enter/exit transitions. These can be explicit patterns like `slide-left` (does a translation in the specified direction) which provide convenient defaults for common use-cases. For instance, sliding a header/footer offscreen. Or higher level abstractions like `previous`/`next` which allow the UA to decide the appropriate animation (for example based on the idiomatic pattern for the native OS).
+
+* `duration` : The total length for this transition.
+
+* `delay` : A Document transition establishes a universal timeline for animations spanning multiple elements. This field indicates the delay in starting animations for this transition on this universal timeline.
+
+* `element` : A reference to the [Element](https://dom.spec.whatwg.org/#interface-element) in the new Document.
+
+* `sharedElement` : A reference to an element in the previous Document, explained in Referencing Elements (#referencing-elements).
+
+If one of `sharedElement` or `element` is set, this is an enter or exit transition respectively. The animations for each animatable property are defined based on `transitionType`. If both `sharedElement` and `element` are set, this is a paired transition. This transition first creates a combined image using painted content from both elements based on the `content` animation specification under [Animatable Properties](#animatable-properties). Other animatable properties are then applied to this combined output.
```

Comment:
>  This transition first creates a combined image using painted content from both elements

I'm not sure what this means. If we have a border-radius 'from' element, and a opacity 0.5 but no border radius 'to' element, then what is the 'combined image' as you describe it here?